### PR TITLE
[stable/drone] add autoscaler deployment

### PR DESCRIPTION
**Pull request by davideagle**
Signed-off-by: Davíð Örn Jóhannsson @davideagle <davideaglephotos@gmail.com>

#### What this PR does / why we need it:
Enables deployment and configuration of Drone Autoscaler
@zakkg3 @paulczar 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/drone/values.yaml [hunk 1]
```diff
@@ -333,6 +341,133 @@ dind:
   #    memory: 2Gi
   #    cpu: 1
 
+autoscaler:
+  enabled: true
```
Comment (zakkg3) [inline stable/drone/values.yaml]: this should be false by default.

**Review by zakkg3 — CHANGES_REQUESTED** [review]
This looks nice.
Can you add some documentation on the readme. on What is this for, how it works, a use case?

**Comment by zakkg3** [comment]
Hey, your last commit failed the test, need a version bump from 2.5.0 to 2.5.1

```
 ✖︎ drone => (version: "2.3.3", path: "stable/drone") > Chart version not ok. Needs a version bump!
```

**Comment by zakkg3** [comment]
have you commited stable/drone/templates/deployment-autoscaler.yaml  ? see no lines.

**Comment by zakkg3** [comment]
/retest
bump the version and solve README conflict so we can merge this.
thanks.

**Comment by christian-roggia** [comment]
This chart is now deprecated, please refer to https://github.com/drone/charts for further support and feature requests.
/close

**Pull request closed without merging**